configs/systemd/inbd.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=INBM Daemon Service
+Requires=network.target
+
+[Service]
+# ExecStart command is only run when everything else has loaded
+Type=idle
+User=root
+EnvironmentFile=-/etc/environment
```

Comment:
none of the `Environment` directives are needed yet--we can add later if needed

Reply:
All `Environment` directives removed.